src/azure-cli/azure/cli/command_modules/serviceconnector/custom.py

```diff
@@ -263,6 +264,9 @@ def connection_create(cmd, client,  # pylint: disable=too-many-locals disable=to
                                      'manually and then create the connection.'.format(str(e)))
 
     validate_service_state(parameters)
+    new_auth_info = enable_mi_for_db_linker(cmd, source_id, target_id, auth_info, source_type, target_type, client_type,
```

Comment:
the params of the function sames to be duplicated, e.g, source_id and source_type, target_id and target_type, can we reuse them?

Reply:
Updated